pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeTableDataManager.java

```diff
@@ -51,7 +51,7 @@
 import org.apache.pinot.core.realtime.impl.RealtimeSegmentStatsHistory;
 import org.apache.pinot.core.segment.index.loader.IndexLoadingConfig;
 import org.apache.pinot.core.segment.index.loader.LoaderUtils;
-import org.apache.pinot.core.segment.virtualcolumn.VirtualColumnProviderFactory;
+import org.apache.pinot.core.segment.virtualcolumn.ColumnProviderFactory;
```

Comment:
Can we revert this class name change? Seems it is not used in this pr?

Reply:
It is used in MutableSegmentImpl to create DefaultNullValueColumnProvider, for virtual and the extra columns